### PR TITLE
#2037 removing unused packages

### LINKED ISSUE #2037: Check if we need @vue/cli-plugin family
@vikiival said it's bundled in @nuxt packages tho, let's review and remove unnecessary packages

https://github.com/kodadot/nft-gallery/blob/main/package.json#L109

```json
 "@vue/cli-plugin-babel": "^3.12.1",
    "@vue/cli-plugin-pwa": "^4.5.15",
    "@vue/cli-plugin-typescript": "^4.5.15",
    "@vue/cli-service": "^4.5.15",
```

### Ref
- https://github.com/kodadot/nft-gallery/pull/2035

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Do a quick check before the merge.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR closes #2037
- [x] removed all the unused packages 

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases

### Had issue bounty label ?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI, a screenshot is best to understand changes for others.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -105,10 +103,7 @@
     "@types/webpack": "^4.41.32",
     "@typescript-eslint/eslint-plugin": "^5.10.0",
     "@typescript-eslint/parser": "^5.10.0",
-    "@vue/cli-plugin-babel": "^4.5.15",
-    "@vue/cli-plugin-pwa": "^4.5.15",
     "@vue/cli-plugin-typescript": "^4.5.15",
```
Comment (yangwao) [inline package.json:106]: I guess we are not using this as well?

**Review by yangwao — COMMENTED** [review]
Curious

File: package.json (hunk 1 above)
Comment (prachi00) [reply package.json:106]: `@vue/cli-plugin-typescript` removing this breaks the build, so I think its being used internally for some setup

**Comment by yangwao** [comment]
@prachi00 check why tests has failed https://github.com/kodadot/nft-gallery/runs/4914110706?check_suite_focus=true

**Comment by prachi00** [comment]
@yangwao tests are fixed, good to go now.

**Comment by yangwao** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 167.63 USD/KSM ~ 0.597 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x717cc80f27a16993e25ab7e7a2267094ca342302d2f7db07cd83dbb3433da1d8](https://kusama.subscan.io/extrinsic/0x717cc80f27a16993e25ab7e7a2267094ca342302d2f7db07cd83dbb3433da1d8)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by yangwao** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 157.66 USD/KSM ~ 0.634 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x02c4c6c7923456b5c7831105f926b27563bc1d3fbfa15dd540a9b60369dfdda2](https://kusama.subscan.io/extrinsic/0x02c4c6c7923456b5c7831105f926b27563bc1d3fbfa15dd540a9b60369dfdda2)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**